openslides_backend/action/relations/relation_manager.py

```diff
@@ -44,9 +40,6 @@ def get_relation_updates(
         # id has to be provided to be able to correctly update relations
         assert "id" in instance
 
-        if not process_calculated_fields_only:
```

Comment:
The `process_calculated_fields_only` seems to be dysfunctional now. Please remove it, if is not needed anymore.

Reply:
It is removed